examples/benchmark/benchmark_fbopt_pacs_full.yaml

```diff
@@ -100,6 +106,7 @@ diva_fbopt:
     - msel_tr_loss
     - msel
     - es
+    - gamma_d
```

Comment:
depending on whether we register gamma_d to fbopt or not:  if we register, we do not need to tune gamma_d here.  Maybe we could tune the intiial value for gamma_d, since the intial value for all dynamically adjusted hyperparameters might be vital